frontend/packages/kubevirt-plugin/integration-tests/tests/vm.overview.scenario.ts

```diff
@@ -123,4 +123,12 @@ describe('Test VM overview', () => {
       );
     });
   });
+
+  it('Check unpause VM when VM is running', async () => {
+    await vm.action(VM_ACTION.Start);
```

Comment:
I think this deserves a docstring. I don't understand why opening a modal window unpauses the VM